.github/workflows/update-codeflare-operator.yml

```diff
@@ -0,0 +1,60 @@
+name: Update CodeFlare operator with latest changes from MCAD
+
+on:
+  workflow_run:
+    workflows: [Build and Push dev MCAD image into Quay]
+    types:
+      - completed
+
+jobs:
+  update-codeflare-operator:
+    runs-on: ubuntu-latest
+    if: github.event.workflow_run.head_branch == 'main'
```

Comment:
Should it also be checked that the workflow run was not in a fork?

Reply:
Good point, will add it

Reply:
Done, added a condition to check repository name and make sure that only successful dev image build trigger the workflow.